openmetadata-ui/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -714,6 +723,10 @@ const DatasetDetailsPage: FunctionComponent = () => {
           (d) => d.testCase.tableTestType !== testType
         );
         setTableTestCase(updatedTest);
+        showToast({
+          variant: 'success',
+          body: `Test successfully deleted.`,
```

Comment:
```suggestion
          body: `Test deleted successfully`,
```
Also, move this out to constant and use it from there.